CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+## [0.5.1] - [December 28, 2020]
+* Add two properties to set the forward chevron and color in iOS
```

Comment:
I think it could be something like "Added `iosChevron` and `iosChevronPadding` for forward chevron and color in iOS".

It'll make it clearer for someone reading the changelog